eventhubs/client/azure-eventhubs/src/samples/java/ReceiveEventsFromKnownSequenceNumberPosition.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This scenario is pretty confusing. I'm having a hard time following it. I think you can express it easier by:
1. Getting partition properties
1. Getting the lastEnqueuedSequenceNumber from PartitionProperties
1. Creating a consumer from that sequenceNumber.
1. `.receive().take(1)`